virttest/qemu_vm.py

```diff
@@ -993,8 +993,13 @@ def add_memorys(devices, params):
             output = re.findall(r",memory-backend=mem-([\w|-]+)", machine_cmd)
             if output:
                 name = output[0]
-                backend_options = {}
+                backend_options = params.object_params("machine_mem")
                 backend_options["size"] = "%sM" % params["mem"]
+                for value in backend_options:
+                    if value in ['yes', 'on', True]:
+                        backend_options[value] = "on"
```

Comment:
Would you pls explain why we need convert the value? Seems all of 'yes, on, true' works in qemu cli.

Reply:
thanks a lot, corrected.